### ISSUE TITLE
Running ChexAlign on parallel

### ISSUE THREAD

**Issue by sga91** [seqcode/chexalign#1]
Is there a way to speed up the alignments using multiple cores?

Thanks!

**Comment by naomiyamada** [comment]
Yes, I've been wanting to make it multi-threading. I'll try to find the time for it in November.
